client/js/transports/small-webrtc.mdx

```diff
@@ -0,0 +1,169 @@
+---
+title: "SmallWebRTCTransport"
+description: "A lightweight WebRTC transport for peer-to-peer connections with Pipecat"
+---
+
+The `SmallWebRTCTransport` class provides a WebRTC transport layer for connecting to Pipecat's `SmallWebRTCTransport` server-side implementation. It handles audio/video device management, WebRTC connections, and real-time communication between clients and bots.
+
+## Usage
+
+### Basic Setup
+
+```javascript
+import { RTVIClient } from "@pipecat-ai/client-js";
+import { SmallWebRTCTransport } from "@pipecat-ai/small-webrtc-transport";
+
+const transport = new SmallWebRTCTransport();
+
+const rtviClient = new RTVIClient({
+  transport,
+  enableCam: false, // Default camera off
+  enableMic: true, // Default microphone on
+  callbacks: {
+    // Event handlers
+  },
+  params: {
+    baseUrl: "http://localhost:7860",
+    endpoints: {
+      connect: "/api/offer",
+    },
+  },
+});
+
+await rtviClient.connect();
+```
+
+## API Reference
+
+### Constructor
+
+```typescript
+new SmallWebRTCTransport();
+```
+
+No parameters are required for basic functionality.
+
+### Properties
+
+<ResponseField name="iceServers" type="string[]">
+  Array of STUN/TURN server URLs for ICE connection establishment. Default is `["stun:stun.l.google.com:19302"]`.
+
+```javascript
+// Set custom ICE servers
+transport.iceServers = [
+  "stun:stun.l.google.com:19302",
+  "stun:stun1.l.google.com:19302",
+];
+```
+
+</ResponseField>
+
+### Methods
+
+Most methods have an equivalent in the RTVI client. The transport is designed to be used with the RTVI client, so in most cases you should use the RTVI client methods to interact with the transport, rather than calling these methods directly. This is especially true for `initialize()`, `connect()`, and `disconnect()`.
+
+<ResponseField name="setAudioCodec" type="method">
+  Sets the preferred audio codec.
+
+```javascript
+transport.setAudioCodec("opus");
+```
+
+</ResponseField>
+
+<ResponseField name="setVideoCodec" type="method">
+  Sets the preferred video codec.
+
+```javascript
+transport.setVideoCodec("VP8");
+```
+
+</ResponseField>
+
+### States
+
+The transport can be in one of these states:
+
+```typescript
+type TransportState =
+  | "initializing"
+  | "initialized"
+  | "connecting"
+  | "connected"
+  | "ready"
+  | "disconnecting"
+  | "disconnected"
+  | "error";
+```
+
+## Events
+
+The transport implements the various [RTVI event handlers](https://docs.pipecat.ai/client/js/api-reference/callbacks). These events will fire when used with an RTVIClient.
+
+Key callback events:
```

Comment:
what about `onBotReady`/`onClientReady`?

Reply:
also. random aside: what does it mean to [hydrate your application](https://linear.app/dailyco/issue/CSDK-2674/canreceive-permission-react-hooks-update)? is this something where i'm going to find out that i'm old and out of touch with the cool kids' lingo?